src/dex/fluid-dex/fluid-dex.ts

```diff
@@ -892,6 +897,7 @@ export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
       fee,
       outDecimals,
       currentLimits,
+      centerPrice,
       syncTime,
     );
 
```

Comment:
should we update `if (amountIn === 2n ** 256n - 1n)` to `if (amountIn === 0n) {` if we decided to return `0n` instead of `2n ** 256n - 1n` ?

Reply:
I think it's just dead code, I'll remove the check, cause if we return 0 from swapOutAdjusted, we'll return 0 from swapOut, cause other params don't matter, we'll zero out anyways

Reply:
but let me check first

Reply:
yeah, max uint should not be returned anyway, but this check doesn't hurt

Reply:
ok, but this check just doesn't make sense anyway

Reply:
removed